tools/deploy.py

```diff
@@ -0,0 +1,27 @@
+"""Deploys a validated version of All of Us terminology to the RDR."""
+
+import logging
+
+from client import Client
+from main_util import configure_logging, get_parser
```

Comment:
I'm seeing a few things I want out of the RDR codebase. That could imply:
* this repo includes some RDR code (git clones it automatically? git submodule? we factor it out?)
* we define the JSON validation code in RDR, and this repo calls some of those functions

It seems a little crazy to depend on the whole RDR codebase from here, though.

For `main_util` we could easily just copy files, and I don't feel too bad about that.

For `client.Client`, it seems bad to reimplement the auth, so that seems like the firmer constraint on some code sharing.